ideasbox/templates/ideasbox/pagination.html

```diff
@@ -2,13 +2,24 @@
 {% if is_paginated %}
     <div class="pagination">
         {% if page_obj.has_previous %}
-            <a href="{{ base_url }}?page={{ page_obj.previous_page_number }}" class="previous">{% trans "previous" %}</a>
+
+            {% if page_obj.previous_page_number != 1 %}
```

Comment:
python manage.py test 
does not reproduce the error. How can I produce the failing test on my side please ?